tests/vmi_lifecycle_test.go

```diff
@@ -485,6 +485,37 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 			})
 		})
 
+		Context("when guest crashes", Label("crashing-guest"), decorators.VMIlifecycle, func() {
+			It("should be stopped and have Failed phase when a PanicDevice is provided", func() {
+				vmi := libvmifact.NewFedora(libvmi.WithPanicDevice(v1.Isa))
+				vmi, err := kubevirt.Client().VirtualMachineInstance(testsuite.GetTestNamespace(vmi)).Create(context.Background(), vmi, metav1.CreateOptions{})
+				Expect(err).ToNot(HaveOccurred(), "Should create VMI successfully")
+				libwait.WaitUntilVMIReady(vmi, console.LoginToFedora)
+
+				ctx, cancel := context.WithCancel(context.Background())
+				defer cancel()
+
+				By("Crashing the vm guest")
+				Expect(console.SafeExpectBatch(vmi, []expect.Batcher{
+					&expect.BSnd{S: "sudo su -\n"},
+					&expect.BExp{R: "#"},
+					&expect.BSnd{S: `echo c > /proc/sysrq-trigger` + "\n"},
+					&expect.BExp{R: "#"},
+				}, 10)).To(Succeed())
+
+				By("Waiting for the vm to be stopped")
+				event := watcher.New(vmi).SinceWatchedObjectResourceVersion().Timeout(15*time.Second).WaitFor(ctx, watcher.WarningEvent, v1.Stopped)
+				Expect(event.Message).To(ContainSubstring(`The VirtualMachineInstance crashed`), "VMI should be stopped because of a guest crash")
+
+				By("Checking that VirtualMachineInstance has 'Failed' phase")
+				Eventually(func() v1.VirtualMachineInstancePhase {
+					vmi, err := kubevirt.Client().VirtualMachineInstance(vmi.Namespace).Get(context.Background(), vmi.Name, metav1.GetOptions{})
+					Expect(err).ToNot(HaveOccurred(), "Should get VMI successfully")
+					return vmi.Status.Phase
+				}, 10, 1).Should(Equal(v1.Failed), "VMI should be failed")
```

Comment:
`Eventually(matcher.ThisVMI(vmi)).WithTimeout(10*time.Second).WithPolling(time.Second).Should(matcher.BeInPhase(v1.Failed))`